client/src/services/api/scientific-evidence-stats.ts

```diff
@@ -8,7 +8,16 @@ export const AXIOS_INSTANCE = Axios.create({
 
 export const ScientificEvidenceStatsAPI = <T>(config: AxiosRequestConfig): Promise<T> => {
   const source = Axios.CancelToken.source();
-  const promise = AXIOS_INSTANCE({ ...config, cancelToken: source.token }).then(({ data }) => data);
+  const promise = AXIOS_INSTANCE({ ...config, cancelToken: source.token })
+    .then(({ data }) => data)
+    .catch((error) => {
+      if (Axios.isAxiosError(error) && error.response?.status === 403) {
```

Comment:
this is too specific. If you have the incorrect value, odds are you will get a 404, not a 404. Please make it generic enough so that it handles all error cases